logger.go

```diff
@@ -187,47 +187,46 @@ func (log *Logger) With(fields ...Field) *Logger {
 // is enabled. It's a completely optional optimization; in high-performance
 // applications, Check can help avoid allocating a slice to hold fields.
 func (log *Logger) Check(lvl zapcore.Level, msg string) *zapcore.CheckedEntry {
-	return log.check(lvl, msg)
+	return log.check(lvl, msg, 0)
+}
+
+func (log *Logger) log(lvl zapcore.Level, msg string, skip int, fields ...Field) {
+	if ce := log.check(lvl, msg, skip); ce != nil {
+		ce.Write(fields...)
+	}
```

Comment:
nit: Let's move `Logger.log` below the exported definitions (maybe right above or below `Logger.check`).